### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by Alena1508**

### CODE REVIEW

File: src/Login/Login.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  render() {
+    const { loading } = this.state;
+    return (
+      loading ? <Loader /> :
+        <form onSubmit={(e) => this.submit(e)}>
+        <input type="text" placeholder="Name"
+               name="name" required/>
+        <input type="password" placeholder="Password"
+               required/>
```
Comment (iliatcymbal) [inline src/Login/Login.js]: why attributes on the same line?

File: src/NotFound/NotFound.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+export class NotFound extends React.Component {
+  constructor(props) {
+    super(props);
+  }
```
Comment (iliatcymbal) [inline src/NotFound/NotFound.js]: no need to invoke empty constructor

File: src/NotFound/NotFound.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+
+  render() {
+
+    return (
+      <p>Ooops...!!! This page not found!</p>
+    )
+  }
+}
```
Comment (iliatcymbal) [inline src/NotFound/NotFound.js]: I would make it as pure component, you don't need any life cycles or state here

File: src/Logined/Logined.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+          <Route
+            path="*"
+            component={NotFound}
+          />
+          <Route render={({location}) => <h1>Page <em> {location.pathname.replace('/', '')}</em> is not exist</h1>}
+          />
+        </Switch>
+      )
+  }
+}
```
Comment (iliatcymbal) [inline src/Logined/Logined.js]: I think you can use there pure component

File: src/TaskList/TaskList.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+        "description": "watch King Lion, How to Train Your Dragon",
+        "title": "Watch cartoons",
+        "day": 6,
+        "id": "60"
+      }
+    ]
+  ];
+
+  return (
+    <Tabs>
```
Comment (iliatcymbal) [inline src/TaskList/TaskList.js]: why the current day is not selected?

**Review by iliatcymbal — COMMENTED** [review]
Please, review my notes and let me know when it be ready

**Comment by Alena1508** [comment]
Fixed bugs

**Comment by iliatcymbal** [comment]
Good, make merge

**Pull request merged**